endpoints/getting-started/openapi.yaml

```diff
@@ -6,6 +6,9 @@ info:
   version: "1.0.0"
 host: "echo-api.endpoints.YOUR-PROJECT-ID.cloud.goog"
 # [END swagger]
+# For App Engine deployments, delete the above "host:" line and remove the "# "
```

Comment:
Yes, that's correct. The *.appspot.com is used only on App Engine, whereas the other form is used for GCE and GKE.